### ISSUE TITLE
wiki_ours2

### ISSUE THREAD

**Issue by zyfsa** [devraj89/LABEL-CONSISTENT-MATRIX-FACTORIZATION-BASED-HASHING-FOR-CROSS-MODAL-RETRIEVAL#1]
hi，I have a question about the sampleInds = sampleInds(1:N). where I can find it. Now I want to train my own dataset

**Comment by devraj89** [comment]
Hi

Thanks for your interest in our work. So basically the sampleInds was used to select a random number of samples to constitute the training dataset. As far as I remember for MirFlickr and NUS-Wide sampleInds was a vector of 5000 numbers. So when we did `sampleInds = sampleInds(1:N)` we were just selecting the data for getting the training set.

For your own dataset suppose you want to select `N` number of samples for your training 
simply do this 
`sampleInds = 1:N`
I hope this clears your doubt. Please get in touch with me in case of any further issues.

**Comment by 152ls12** [comment]
Hi
error:Undefined function 'sampleInds' for input arguments of type 'double',what should I do.
